### PR TITLE
CVE-2019-19338 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/19xxx/CVE-2019-19338.json [hunk 1]
```diff
@@ -4,15 +4,82 @@
...
+                "url": "https://www.openwall.com/lists/oss-security/2019/12/10/3"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A flaw was found in the fix for CVE-2019-11135, the way Intel CPUs handle speculative execution of instructions when a TSX Asynchronous Abort (TAA) error occurs. When a guest is running on a host CPU affected by the TAA flaw (TAA_NO=0), but is not affected by the MDS issue (MDS_NO=1), the guest was to clear the affected buffers by using a VERW instruction\r\nmechanism. But when the MDS_NO=1 bit was exported to the guests, the guests did not use the VERW mechanism to clear the affected buffers. This issue affects guests running on Cascade Lake CPUs and requires that host has 'TSX' enabled. Confidentiality of data is the highest threat associated with this vulnerability."
```
Comment (iamleot) [inline 2019/19xxx/CVE-2019-19338.json]: Please mention product vendor and name in the description (i.e. `Linux kernel`).

**Comment by ccoffin** [comment]
Red Hat CNA,

As was previously requested by you, we are currently waiting for the iamleot changes to be made before populating this CVE entry. Let us know if you need anything or have any questions.

Regards,

Chris

**Comment by cristina479** [comment]
Red Hat CNA,

Can you provide an affected or fixed version for this CVE ID in order to publish it to the CVE List?

Thank you,
Cristina

**Comment by cve-team** [comment]
Please provide an affected or fixed version in the version_data and in the description.
Thank you.

**Comment by cristina479** [comment]
Dear darunesh-RH,

Can you include the requested changes to the `version_data` and `description` fields?

**Comment by tausif-rh** [comment]
> Please provide an affected or fixed version in the version_data and in the description.
> Thank you.

Analysis just confirmed the fixed-in version information:

$ git tag --contains cbbaa2727aa3ae9e0a844803da7cef7fd3b94f2b
$ git tag --contains c11f83e0626bdc2b6c550fc8b9b6eeefbd8cefaa
$ git tag --contains b07a5c53d42a8c87b208614129e947dd2338ff9c
v5.5
v5.6
v5.7
...

This issue is fixed since upstream kernel v5.5.

https://github.com/CVEProject/cvelist/pull/3122/files#diff-dc6d1b6b2f906131e021f0c8e35822a5

**Pull request merged**